### DISCUSSION TITLE
Problemas ao importar arquivo server.json do Curso 7188

### DISCUSSION THREAD

**Discussion by lxxzh** [balta-io/forum#247] [Tira Dúvidas] [answered]
Boa noite @andrebaltieri , sabe informar se o arquivo server.json do curso 7188 está corrompido? Pois estou tentando importar no Mockoon e não aparece nada.

<img width="958" alt="image" src="https://user-images.githubusercontent.com/106619427/216244410-20f7c979-21d5-4e4a-81ea-2353262e5737.png">

<img width="955" alt="image" src="https://user-images.githubusercontent.com/106619427/216244456-c5c34b31-0006-4b82-8cc9-194dac88a5cb.png">

<img width="955" alt="image" src="https://user-images.githubusercontent.com/106619427/216244498-ea1ab6f2-d808-4022-9443-270aef269e7f.png">

**Importando um outro arquivo JSON do curso 7181, consigo abrir normalmente, segue o exemplo abaixo:** 

<img width="952" alt="image" src="https://user-images.githubusercontent.com/106619427/216244715-46cfb077-639a-41b8-8223-d41ca5c8936a.png">

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @lxxzh, aqui tem a solução:
https://github.com/balta-io/7188/discussions/26

**Reply by lxxzh** [reply]
Não consegui a solução, pelo que vi são arquivos / cursos diferentes do qual estou relatando, no último print que mencionei acima foi só um teste com outro arquivo JSON do curso 7181 que deu certo!  Mas o JSON deste curso 7188 não deu.


Eu estou importando pelo "OPEN ENVIRONMENT" conforme a aula.
<img width="419" alt="image" src="https://user-images.githubusercontent.com/106619427/216501815-e61f5aa3-eb11-4455-acf0-590a07cbc2a2.png">

E o arquivo JSON é este aqui: https://github.com/balta-io/7188/blob/master/api/server.json 
Se puder simular ai e ver oque aparece.

**Reply by andrebaltieri** [reply]
Boa tarde, @lxxzh, como vai?
Qual erro deu? Você seguiu os passos que mostrei no link? Tem um vídeo lá mostrando como fiz para importar a nova versão.

**Reply by lxxzh** [reply]
Boa tarde @andrebaltieri , tranquilo e contigo? Segui os passos do link, mas sem sucesso! 

O erro é não aparecer os dados no Mockoon como este da aula. 
<img width="671" alt="image" src="https://user-images.githubusercontent.com/106619427/216659692-20a36846-295b-45ce-bcff-47a928eea3b7.png">


Estou fazendo assim: 
<img width="949" alt="image" src="https://user-images.githubusercontent.com/106619427/216658016-160c533c-4cd0-435e-a5a8-f0f3177b0b58.png">
<img width="954" alt="image" src="https://user-images.githubusercontent.com/106619427/216658185-49a69d51-ca25-4b79-a560-152b96518ee4.png">
<img width="953" alt="image" src="https://user-images.githubusercontent.com/106619427/216658219-90d8d63b-731b-41e7-9907-00f9704005d4.png">
<img width="955" alt="image" src="https://user-images.githubusercontent.com/106619427/216658268-ffcf6eb5-66eb-470a-a925-87b3545fbae6.png">

**Este JSON que estou tentando importar é do curso Criando Apps Web com Angular 8 e Ionic 4, da página de administração do PETSHOP**

**Reply by andrebaltieri** [reply]
Bom dia, @lxxzh como vai?
Você tentou importar este arquivo aqui: https://github.com/balta-io/7187/discussions/60#discussioncomment-3439600

**Reply by lxxzh** [reply]
Bom dia @andrebaltieri, tudo beleza e contigo?
 Sim importei este arquivo e da certo:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/106619427/217009616-65f0a758-d51b-4349-beba-1dffdc95bb45.png">

**Mas acontece que o arquivo JSON da aula não é esse.**

Olha o arquivo JSON da aula "STARTUP DO PROJETO" do curso Criando apps Web com Angular 8 e IONIC 4
<img width="935" alt="image" src="https://user-images.githubusercontent.com/106619427/217010478-c4565fba-74e8-4c83-b904-65cc8704565c.png">

**Reply by andrebaltieri** [reply]
@lxxzh, boa tarde.
Refiz o arquivo para versão mais nova do Mockoon...
https://github.com/balta-io/7188/blob/master/api/7188_updated-2023-02-06.json

**Reply by lxxzh** [reply]
Boa tarde Balta, show de bola, agora sim deu certo. Muito Obrigado!
<img width="954" alt="image" src="https://user-images.githubusercontent.com/106619427/217050996-96725245-3ce2-40b6-821f-d20af549632e.png">
